ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -203,6 +203,12 @@ def fetch_tables(
                         "Table pattern not allowed",
                     )
                     continue
+                if self._is_partition(table_name, schema):
```

Comment:
@igor-kramer don't we need all the other connectors implementing this method and atleast doing a pass otherwise might fail for other connectors?

Reply:
never mind, didn't see the below code

Reply:
actually we might just be able to just define it in `SqlSource` with a `False` and let specific connectors override the definition with their own implementation when needed. I believe the current approach should work